Misc/NEWS.d/next/Core and Builtins/2019-06-05-09-24-17.bpo-37160.O3IAY3.rst

```diff
@@ -0,0 +1,2 @@
+Thread native id support
+- :func:`threading.get_native_id` now also supports NetBSD.

```

Comment:
I don't think that the NEWS entry written as two lines will be rendered properly. My suggestion was to only keep the second sentence: remove " Thread native id support" and remove "- " from the second line.

Reply:
All right sorry will remove it.